build.gradle

```diff
@@ -821,6 +824,7 @@ project('spring-xd-module') {
 		compile "org.springframework.boot:spring-boot-loader"
 		compile "com.fasterxml.jackson.core:jackson-databind"
 		compile project(':spring-xd-module-spi')
+		compile project(':spring-xd-spark-streaming')
```

Comment:
The reason we do that here is because of the ModuleFactory requires to know about `SparkStreamingDriverModule`. Once we address that we can eliminate this dependency.